### PR TITLE
Update hardcoded rids for targeting/runtime pks

### PR DESCRIPTION
**Pull request by ViktorHofer**
The rids in the 6.0 Preview 1 SDK differ to the hardcoded ones in our repo which caused CLI errors locally.

### CODE REVIEW

File: eng/targetingpacks.targets [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <Project>
+  <PropertyGroup>
+    <SharedFrameworkName Condition="'$(SharedFrameworkName)' == ''">Microsoft.NETCore.App</SharedFrameworkName>
```
Comment (eerhardt) [inline eng/targetingpacks.targets:3]: (minor side question): When/why is this necessary?

**Review by eerhardt — APPROVED** [review]
Just a little question. LGTM

File: eng/targetingpacks.targets (hunk 1 above)
Comment (ViktorHofer) [reply eng/targetingpacks.targets:3]: I'm intending to use this file for the linker infrastructure to avoid the duplication of the runtime pack bootstrapping logic. As I was already at it, I defaulted the property so that there's one less that needs to passed in globally when building the linker tests.

**Comment by ViktorHofer** [comment]
@safern @hoyosjs looks like there's a race which causes the runtime tests leg trying to download assets that haven't yet been published?

**Comment by safern** [comment]
> @safern @hoyosjs looks like there's a race which causes the runtime tests leg trying to download assets that haven't yet been published?

Yeah, that's a known issue. That should require: https://github.com/dotnet/runtime/issues/43952 in order to be fixed, or calculate the set of dependencies correctly for Mono runtime tests. @naricc do you have an update on that issue?

File: eng/targetingpacks.targets [hunk 2]
```diff
@@ -15,15 +19,15 @@
                              LatestRuntimeFrameworkVersion="$(ProductVersion)"
                              RuntimeFrameworkName="$(SharedFrameworkName)"
                              RuntimePackNamePatterns="$(SharedFrameworkName).Runtime.**RID**"
-                             RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;ios-arm64;ios-arm;ios-x64;ios-x86;tvos-arm64;tvos-x64;android-arm64;android-arm;android-x64;android-x86;browser-wasm"
+                             RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;linux-musl-arm;ios-arm64;ios-arm;ios-x64;ios-x86;tvos-arm64;tvos-x64;android-arm64;android-arm;android-x64;android-x86;browser-wasm;osx-arm64"
```
Comment (safern) [inline eng/targetingpacks.targets:22]: Should the SDK have a property that exposes the RIDs that it supports?

**Comment by naricc** [comment]
@safern As we discussed in the issue, we decided the best way to do it build a corerun that uses mono. I will be working on that next week.
